javascript/src/main/components/Students/StudentTable.js

```diff
@@ -19,14 +19,23 @@ const StudentTable = ({students,deleteStudent}) => {
     } 
```

Comment:
The PR description says:

> In this PR, I can upload a file in this format, and the table is populated and if the file has an incorrect format, some reasonable error is displayed.

I have so many questions:
* What format?
* On what page in the app?
* What table is populated?

The PR description should be clear enough that I am not left wondering.   Without knowing the answers, a code reviewer is in no position to review the code or test.    They shouldn't have to go looking through the issue to find answers.

Also the title is currently: 

> 7pm 4 cs vupload 

That's not only too vague, it just has an obvious typo in it.   It's important to take care with these things.  It conveys respect for your fellow team members, and for the work you are doing.

The code looks good to me though, and if it works when I test it, as long as some changes are made to the description and title, this may be mergeable.